rkt/fetch.go

```diff
@@ -27,12 +28,12 @@ import (
 )
 
 const (
-	defaultOS   = runtime.GOOS
-	defaultArch = runtime.GOARCH
+	defaultOS = runtime.GOOS
```

Comment:
I can do that, but it will be a simple "return runtime.GOOS" functions, so useless except that it looks similar to GetArch().